tests/imageresize.js

```diff
@@ -435,6 +439,92 @@ describe( 'ImageResize', () => {
 		}
 	} );
 
+	describe( 'percent resizing', () => {
+		describe( 'standard image', () => {
+			before( () => {
+				customConfig = CONFIG_RESIZE_IN_PIXELS;
```

Comment:
You mean in percentage? Pixels are the default in this test file.

Reply:
BTW, we tend to simply repeat `.create()` calls in each test file instead of deduplicating it like this. It's easier to read the code if it's closer to you and doesn't have conditions inside.

Reply:
I'm not entirely sure about this one, it introduces notable code repetition across the file. Added changes in [`t/306b`](https://github.com/ckeditor/ckeditor5-image/tree/t/306b) branch, can be compared to t/306.

Reply:
Tests are all about readability. If you need to jump to 10 places to understand what a single TC does, you have a problem. Therefore, repetition is better. Or rather – keeping as much code locally in a TC is better.